tools/doc/html.js

```diff
@@ -8,6 +8,15 @@ var typeParser = require('./type-parser.js');
 
 module.exports = toHTML;
 
+// customized heading without id attribute
+var renderer = new marked.Renderer();
```

Comment:
It doesn't matter. I rebased upstream and resolved the conflict. 
I just changed the template to string add before I saw your latest comment :D
I think either is OK. At least the style is consistent now. We can es6ify it later.